terraform/main.tf

```diff
@@ -0,0 +1,35 @@
+module "devops_knowledgebase" {
+    source = "./modules/knowledgebases"
+}
+
+module "devops_knowledge_documents" {
+    source = "./modules/knowledge-documents"
+    knowledgebase_id = module.devops_knowledgebase.knowledgebase_id
+}
+
+module "flows" {
+    depends_on = [
+      module.devops_knowledgebase
+    ]
+    source = "./modules/flows"
+}
+
+data "genesyscloud_flow" "devops_inbound_flow" {
```

Comment:
Just out of curiosity why are you looking up the flow id.  If you are deploying the flow via a module, you can create an output variable in the module and explicitly export the flow id from the module.  This avoids an extra call and is the more idiomatic way of doing this.  This also means you can get rid  of the depends on wherever you use the flow id because Terraform will resolve the dependency based on what other modules use that exported flow id.

Reply:
Good point